spec/model/post_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  describe 'validates' do
+    subject { Post.new title: 'Post 1', user_id: 0 }
+
+    before { subject.save }
+
+    it 'should have a title' do
+      subject.title = nil
+      expect(subject).to_not be_valid
+    end
+
+    it 'should not have a blank title' do
+      subject.title = ' '
+      expect(subject).to_not be_valid
```

Comment:
## OPTIONAL 
* Since this error wasn't mentioned by the previous reviewers am making this an optional suggestion but i would strongly advise you to fix this issue in your free time as you have some tests failing 
![image](https://user-images.githubusercontent.com/52497246/145701300-6820826d-2dbe-4c00-8bce-a7d93b1efd5f.png)